<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>form_ejercicio</class>
 <widget class="QWidget" name="form_ejercicio">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>712</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Ejercicio</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(3, 3, 3);
background-color: rgb(44, 43, 58);
font: 10pt &quot;Liberation Sans&quot;;
color: rgb(252, 252, 252);
selection-background-color: rgb(155, 155, 185);</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(44, 43, 58);




</string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Ejercicio</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="2" column="0">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QTableWidget" name="tw_detalle_ejerc">
           <property name="styleSheet">
            <string notr="true">font: 12pt &quot;Liberation Sans&quot;;
</string>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>195</number>
           </attribute>
           <column>
            <property name="text">
             <string>N°</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Mes</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Estado</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_cerrar">
           <property name="text">
            <string>Cerrar</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="minimumSize">
                <size>
                 <width>75</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>Nombre: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="ln_nombre">
               <property name="minimumSize">
                <size>
                 <width>300</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>300</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer_2">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="label_2">
               <property name="minimumSize">
                <size>
                 <width>75</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>75</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="text">
                <string>CUIT / CUIL: </string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="lne_cuit">
               <property name="minimumSize">
                <size>
                 <width>150</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>150</width>
                 <height>16777215</height>
                </size>
               </property>
               <property name="styleSheet">
                <string notr="true">
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="horizontalSpacer">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QPushButton" name="btn_buscar">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(64, 63, 78);</string>
               </property>
               <property name="text">
                <string>Buscar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_actualizar">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(64, 63, 78);</string>
               </property>
               <property name="text">
                <string>Actualizar</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QPushButton" name="btn_eliminar">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(64, 63, 78);</string>
               </property>
               <property name="text">
                <string>Eliminar</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_nuevo">
               <property name="styleSheet">
                <string notr="true">background-color: rgb(64, 63, 78);</string>
               </property>
               <property name="text">
                <string>Nuevo</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <widget class="QTableWidget" name="tw_ejercicio">
         <property name="styleSheet">
          <string notr="true">font: 12pt &quot;Liberation Sans&quot;;
</string>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>195</number>
         </attribute>
         <column>
          <property name="text">
           <string>Año</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Descripción</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tipo</string>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>ln_nombre</tabstop>
  <tabstop>lne_cuit</tabstop>
  <tabstop>btn_buscar</tabstop>
  <tabstop>btn_actualizar</tabstop>
  <tabstop>btn_eliminar</tabstop>
  <tabstop>btn_nuevo</tabstop>
  <tabstop>tw_ejercicio</tabstop>
  <tabstop>tw_detalle_ejerc</tabstop>
  <tabstop>btn_cerrar</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
